repair/row_level.cc

```diff
@@ -1269,12 +1269,14 @@ class repair_meta {
             // and _working_row_buf are ordered.
             _working_row_buf = utils::stall_free_merge(tmp, row_diff,
                  [this] (const repair_row& x, const repair_row& y) { return _cmp(x.boundary(), y.boundary()) < 0; });
+            utils::stall_free_clear(tmp);
         }
         if (update_hash_set) {
             _peer_row_hash_sets[node_idx] = boost::copy_range<repair_hash_set>(row_diff |
                     boost::adaptors::transformed([] (repair_row& r) { thread::maybe_yield(); return r.hash(); }));
         }
         do_apply_rows(row_diff, node_idx, update_buf).get();
+        utils::stall_free_clear(row_diff);
```

Comment:
It's better to make do_apply_rows() remove elements instead of having a separate loop to clear it afterwards.

Reply:
Done.